packages/recorder/plugins/amplitude/index.js

```diff
@@ -0,0 +1,108 @@
+import { createAmplitude } from './amplitude';
+
+/**
+ * This object defines the Amplitude plugin.
+ *
+ * @param {object} options - Options to pass to the plugin.
+ * @param {boolean} options.immediateStart - Immediately start emitting the
+ * `amplitudelevels` events, or wait for the recorder to start.
+ * @param {boolean} options.stopAfterRecording - Stop emitting the events after the
+ * recorder stops recording.
+ * information.
+ */
+function AmplitudePlugin({
+  immediateStart = false,
+  stopAfterRecording = true,
+}) {
```

Comment:
This can be written as a regular class. Both the functions defined in the constructor and the functions attached to the prototype can be attached as normal class methods.

Reply:
I assumed that to indeed, I tried that at first and somehow did not work as expected.

For now I am happy with this construct. I do need to create some other plugin(s), I will then retry the class again.

Reply:
please recheck.. i tried (again) and now it is in.....